test/adaptive_load/fake_plugins/fake_input_variable_setter/fake_input_variable_setter.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+import "google/rpc/status.proto";
+
+// Configuration for FakeInputVariableSetter (plugin name: "nighthawk.fake_input_variable_setter")
+// that sets the |connections| field in CommandLineOptions.
+message FakeInputVariableSetterConfig {
+  // A factor that multiplies the input value before applying it, in order to test the propagation
+  // of both the input and the config. Optional, default 1.
+  int32 adjustment_factor = 1;
```

Comment:
This is independent of initial_value, but it could really go either way. The only purpose of this field is to confirm the propagation of the information, and an integer is sufficient for that. It's entirely a question of what seems more natural. What do you think?